server.bat

```diff
@@ -0,0 +1 @@
+python server.py
```

Comment:
probably a loading a texture will need it? I am not sure..

On Sat, Dec 19, 2015 at 9:08 PM, Trung Le <notifications@github.com> wrote:

> In server.bat
> <https://github.com/WebGLSamples/WebGL2Samples/pull/3#discussion_r48098851>
> :
>
> > @@ -0,0 +1 @@
> > +python server.py
>
> I don't think we need to run a server here, but let me know otherwise. I'm
> planning on removing them.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WebGLSamples/WebGL2Samples/pull/3/files#r48098851>.
>